phpunit.xml.dist

```diff
@@ -206,8 +206,16 @@
 	</testsuite>
 
 	<testsuite name="paratest-teradata">
+		<!--BUCKET-->
 		<file>tests/Backend/CommonPart1/BucketsTest.php</file>
 		<file>tests/Backend/CommonPart1/CreateTableTest.php</file>
+		<file>tests/Backend/Export/ExportSimpleTest.php</file>
+		<file>tests/Backend/CommonPart2/SnapshottingTest.php</file>
```

Comment:
Tady ten test ti jede? initTestTable používá PK takže to jet nemože.

Reply:
Lokalne mi nefunguje a tag jsem jeste nepustil...

Reply:
nahradil jsem ho novym `tests/Backend/Export/SnapshotSimpleTest.php`